apps/server/src/core/error/utils/error.utils.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpException } from '@nestjs/common';
+import { BusinessError } from '@shared/common';
+import { FeathersError } from '../interface';
+
+export class ErrorUtils {
+	static isFeathersError(error: Error): error is FeathersError {
+		let isFeathersError = false;
+
+		if (error && 'type' in error) {
```

Comment:
Can error be undefined here? That `error &&` is needed`? Also in L.10: Is the question mark needed?

Reply:
Ich denke JS mäßig sollte man das auch sicher stellen. Ich vermute allerdings das ts besser mit error && errror.type klar kommt und das ? bei ?.type dann nicht mehr notwendig ist.